packages/react-components/react-toast/stories/Toast/ToastLifecycle.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import {
+  Toaster,
+  useToastController,
+  Toast,
+  ToastTitle,
+  ToastBody,
+  ToastFooter,
+  ToastStatus,
+} from '@fluentui/react-toast';
+import { useId, Link, Button, Text } from '@fluentui/react-components';
+
+export const ToastLifecycle = () => {
+  const toasterId = useId('toaster');
+  const { dispatchToast } = useToastController(toasterId);
+  const [status, setStatus] = React.useState<ToastStatus>('removed');
+  const toastExists = status !== 'removed';
+  const notify = () => {
+    if (toastExists) {
+      return;
+    }
+
+    dispatchToast(
+      <Toast>
+        <ToastTitle intent="success" action={<Link>Undo</Link>}>
+          Email sent
+        </ToastTitle>
+        <ToastBody subtitle="Subtitle">This is a toast body</ToastBody>
+        <ToastFooter>
+          <Link>Action</Link>
+          <Link>Action</Link>
+        </ToastFooter>
+      </Toast>,
+      { onStatusChange: toastStatus => setStatus(toastStatus) },
+    );
+  };
+
+  return (
+    <>
+      <div>
+        <Text role="status">
+          Current status: <Text weight="bold">{status}</Text>
+        </Text>
```

Comment:
TBH it's not really helpful as users can't see transition of states. Can you please do something similar https://react.semantic-ui.com/modules/checkbox/#usage-dom-comparison? So it will be visible how a toast changes its states.